libs/core/src/lib/wizard/wizard-progress-bar/wizard-progress-bar.directive.ts

```diff
@@ -13,10 +13,29 @@ export type WizardSize = 'sm' | 'md' | 'lg' | 'xl';
         '[class.fd-wizard__progress-bar--xl]': 'size === "xl"'
     }
 })
-export class WizardProgressBarDirective {
+export class WizardProgressBarDirective implements OnInit {
     /**
      * Size of the wizard progress bar.
      */
     @Input()
     size: WizardSize;
+
+    /** @hidden */
+    initDisplay: string;
+
+    /** @hidden */
+    ngOnInit(): void {
+        this.initDisplay = this._elRef.nativeElement.style.display;
+    }
+
+    constructor(private _elRef: ElementRef) {}
+
+    /** @hidden */
+    setCssDisplay(visible: boolean): void {
+        if (visible) {
+            this._elRef.nativeElement.style.display = this.initDisplay;
```

Comment:
You can use `.style.removeProperty("display")` there instead of storing `this.initDisplay`